tests/integration/test_application.py

```diff
@@ -197,6 +197,7 @@ def test_version(capsys, monkeypatch, app):
     assert captured.out == "testcraft 3.14159\n"
 
 
+@pytest.mark.usefixtures("emitter")
```

Comment:
Is it for the debug purposes?

Reply:
It ensures we're not spamming stuff to stdout :-)